coinlib_flutter/android/build.gradle

```diff
@@ -66,6 +76,8 @@ android {
         main {
             jniLibs.srcDirs = ['src/main/jniLibs']
         }
+        main.java.srcDirs += "src/main/kotlin"
+        test.java.srcDirs += "src/test/kotlin"
```

Comment:
Are these two lines necessary? They don't add any existing paths and the build works fine without them.

Reply:
They may not be required. I did not do extensive testing. I added them as they are included when creating a new plugin/lib.

Reply:
OK thanks. I find the example app works OK without these lines, so I'll merge and remove them afterwards. If there are any issues as a result, you can let me know.